explore-analyze/elastic-inference/inference-api.md

```diff
@@ -42,18 +45,24 @@ To add a new interference endpoint using the UI:
 1. Provide the required configuration details.
 1. Select **Save** to create the endpoint.
 
+If your inference endpoint uses a model deployed in Elastic’s infrastructure, such as ELSER, E5, or a model uploaded through Eland, you can configure [adaptive allocations](#adaptive-allocations) to reduce resource usage and save costs.
```

Comment:
It's not just about reducing resources and saving costs, it might be the opposite: spinning up more resources, which increases costs. It all depends on how resource-intensive the operation is. If it demands more computing power, we'll scale up – up until a certain limit. I'd rephrase this sentence, something like, "... you can configure adaptive allocations to dynamically adjust resources based on the demand of current operations", or something along those lines.